test/bats/01_cscli.bats

```diff
@@ -115,13 +115,9 @@ teardown() {
     assert_output "&false"
 
     # complex type
-    rune -0 cscli config show --key Config.PluginConfig
-    assert_output - <<-EOT
-	&csconfig.PluginCfg{
```

Comment:
Actually, the point of this was to test how it pretty-printed a structure, it can be something other than pluginconfig